benchmarks/operator_benchmark/pt/cat_test.py

```diff
@@ -5,48 +5,100 @@
 
 import operator_benchmark as op_bench
 import torch
+import random
 
 
 """Microbenchmarks for Cat operator"""
 
 
 # Configs for PT Cat operator
 cat_configs_short = op_bench.config_list(
-    attr_names=['M', 'N', 'K', 'dim'],
+    attr_names=['sizes', 'N', 'dim'],
```

Comment:
there's currently a perf regression in cat benchmark on cpu that's being investigated (you can look it up in the issues), can you please make sure that the regressed config does not disappear from the benchmark?

Reply:
Sure, there are two configs that show particularly strong regression:

* cat_M128_N1024_K2_dim1_cpu
* cat_M512_N512_K2_dim1_cpu

The latter is still here. I will restore the former.